samples/api/hpp_oit_depth_peeling/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2024, Google
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 the "License";
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+ 
+get_filename_component(FOLDER_NAME ${CMAKE_CURRENT_LIST_DIR} NAME)
+get_filename_component(PARENT_DIR ${CMAKE_CURRENT_LIST_DIR} PATH)
+get_filename_component(CATEGORY_NAME ${PARENT_DIR} NAME)
+
+add_sample(
+    ID ${FOLDER_NAME}
+    CATEGORY ${CATEGORY_NAME}
+	AUTHOR "Google"
+    NAME "hpp_oit_depth_peeling"
```

Comment:
Since this is the name that's displayed on Android, it should be a proper name, see https://github.com/KhronosGroup/Vulkan-Samples/pull/1052/files

Reply:
I think, I don't get what you want here...
I just used the `NAME` from the non-hpp sample and prepended `hpp_` to it.

Reply:
The name should be something that people can read, e.g. "Order independent depth peeling".

I fixed most of those for the existing samples recently.

Reply:
I see... thanks.